src/pages/ecosystem/index.tsx

```diff
@@ -1,14 +1,23 @@
+import { useState } from "react"
+
 import { Box } from "@mui/material"
 
+import Filter from "./Filter"
 import Gallery from "./Gallery"
 import Header from "./Header"
 
 const Ecosystem = () => {
+  const [category, setCategory] = useState("all")
+
+  const handleChangeCategory = value => {
+    setCategory(value)
+  }
+
   return (
     <Box
       sx={{
         position: "relative",
-        maxWidth: "1020px",
+        maxWidth: "1102px",
```

Comment:
why this value?

Reply:
and this is the exact calculated value, or just a rough approximation to work well on some devices?

Reply:
calculated by the width of filter

Reply:
ok, perfect, thx!